src/pages/auth/LoginForm/LoginForm.hoc.tsx

```diff
@@ -20,9 +20,9 @@ export const LoginFormHoc = (props: LoginFormHocProps) => {
     return <LoadingPage />;
   }
 
-  if (!space || !world) {
-    return <NotFoundFallback />;
-  }
-
-  return <LoginForm {...props} space={space} world={world} />;
+  return space && world ? (
```

Comment:
[Question] doesn't it look better if we're rendering the `NotFound` in case any of `(!space || !world)` is invalid first?
so that we're eliminating this ternary and making render more straightforward?

Reply:
It's some times easy to determine, other times harder to figure out what is the "looks better" solution.

In this case I opted to have the `space` and `world` front and more prominent with less syntax clutter than having (! || !) around them.